### PR TITLE
Feature/641 add ability to ask where candidates saw vacancy advertised

### PR DESCRIPTION
**Pull request by mbrookeswebdev**

### CODE REVIEW

File: src/filters.js [hunk 1]
```diff
@@ -149,12 +149,15 @@ const lookup = (value) => {
...
       'irish': 'Irish',
+      'jac-website': 'JAC Website',
       'jewish': 'Jewish',
+      'judicial-office-extranet': 'Judicial Office Extranet',
       'judicial-functions': 'The carrying-out of judicial functions of any court or tribunal',
       'lands-chamber': 'Lands Chamber',
       'leadership-non-legal': 'Leadership - non legal',
       'leadership': 'Leadership',
       'legal': 'Legal',
+      'linkedIn': 'LinkedIn',
```
Comment (warrensearle) [inline src/filters.js:160]: The are usually kebab-case - i.e. `linked-in` rather than `linkedIn`

File: src/views/Apply/AdditionalInformation.vue [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+          <CheckboxItem
+            value="judicial-office-extranet"
+            label="Judicial Office Extranet"
+          />
+          <CheckboxItem
+            value="twitter"
+            label="Twitter"
+          />
+          <CheckboxItem
+            value="linkedIn"
```
Comment (warrensearle) [inline src/views/Apply/AdditionalInformation.vue:48]: See previous comment `linked-in` would fit better with existing kebab-case naming

**Review by warrensearle — CHANGES_REQUESTED** [review]
Just one teeny tiny tweak to make

File: src/filters.js (hunk 1 above)
Comment (mbrookeswebdev) [reply src/filters.js]: I've changed it, and I showed the screenshots to @Rebecca-mcknight1991 , she said it looks good.

File: src/views/Apply/AdditionalInformation.vue (hunk 2 above)
Comment (mbrookeswebdev) [reply src/views/Apply/AdditionalInformation.vue]: Done

**Review by warrensearle — APPROVED** [review]
Great stuff 👍

**Pull request merged**